script.module.ws4py/addon.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.ws4py"
+       name="ws4py"
+       version="0.4.3"
```

Comment:
Upstream is 0.5.1 is this intentional?

Reply:
Yes, since 0.4.3 is the module that I used in script.motr I included the same version to ensure compatibility.
I can play with 0.5.1 later on to see if it works and then update this module to 0.5.1

Reply:
That's fine. Please adjust the license tag for merge.